### PR TITLE
docs: update project maintainers list

### PR DESCRIPTION
**Pull request by GlennBullingham**
## What is the purpose of this PR?

Primarily, this PR seeks to update the MAINTAINERS list, such that it is an accurate representation of active maintainers for the OpenEBS project.  In part, it is a response to [a request for the same](https://github.com/cncf/toc/issues/905) from the CNCF ToC, as part of our ongoing efforts to raise OpenEBS to incubation status.

Secondary to this it aims to:

- simplify the formatting and hence improve readability of the overall document
- remove unnecessary information, including the requirement for reviewers to declare a company affiliation
- adds information considered important, including email contact details for active maintainers 

## Why have I been requested as a reviewer?

If you have been asked to review this PR, it is because you are currently listed in the MAINTAINERS file as an active maintainer of the project.  Therefore, in accordance with good project governance, you are being asked to:

- signify your consent, or otherwise, to the changes herein proposed
- in doing so, signify specifically whether you wish to remain an active maintainer, with associated responsibilities, or otherwise

**If you do not submit a review within the review period (ending 31st Oct 2022), this will be taken to signify your consent to all of the changes to maintainer status proposed in this PR.  If you do not agree, you must respond with either comments or a review to that effect on or before that date**

## Is the published OpenEBS governance policy being followed?

The policy as it currently stands is incomplete, in that it makes no explicit provision for the removal / transition to emeritus status of maintainers who are no longer contactable and thus cannot be contacted to request a self-review of their active status.  Once the active membership of the maintainers group has been established via the route proposed by this PR, those persons will be asked to develop and vote on an enhanced policy document which allows the maintainers to keep this information updated more readily.  That will include establishing the requirement to hold correct contact details for active maintainers.

## What happens next?

This PR will enter a reviewing state.  Comments and changes may be requested, up until the 31st October 2022.  After that time, it will be merged in such state as it is, such changes as have been requested and agreed by its reviewers having been incorporated.

## Other changes

This PR also seeks to update the status of previously listed reviewers.  In this, the criteria for remaining an active reviewer are to a) already be listed as a reviewer in the current version of this file and b) having made at least one PR review for the project in the last 12 months.

Reviewers who have not met these criteria it is being proposed herein to transition to the status of "emeritus reviewer", from where they may be restored to active reviewer status on either their request, or that of another active maintainer or reviewer.  In either case, evidence of continuing active participation in that role will be required.

Signed-off-by: GlennBullingham <glenn.bullingham@datacore.com>

### CODE REVIEW

**Review by AmitKumarDas — APPROVED** [review]

**Review by vishnuitta — DISMISSED** [review]
There is the need for updating maintainers list. Thanks for taking it up.
I would request to make sure that we have mail id and get approvals from the active maintainers list as per the change before we merge this PR.

File: MAINTAINERS
Comment (vishnuitta) [inline MAINTAINERS:3]: I would also request to have a change saying that - only the addition need to be happening to emeritus list but not to be removed.

**Comment by GlennBullingham** [comment]
> There is the need for updating maintainers list. Thanks for taking it up. I would request to make sure that we have mail id and get approvals from the active maintainers list as per the change before we merge this PR.

Hi @vishnuitta, good to hear from you.  Thanks for making contact, and for contributing your review to this PR.  Indeed, one of its objectives is to obtain and hold valid email contact details for the project's active maintainers.  For everyone proposed in that role in going forwards (via the changes on this PR), we hold the details included, which are currently considered to be valid.  If someone wishes to use details different to those currently shown, they should of course feel free to request the appropriate change at any time.

I hope to receive reviews from all those of whom they have been requested, namely, the current maintainer list this repo.  Any accurate and valid changes requested in those reviews, which do not attract majority overrule of the eligible reviewers, will be incorporated prior to it being merged.  However, the default action remains to merge the PR such as it stands at midnight on the 30th Oct.  There cannot be an indefinite period of review, awaiting positive release by all eligible parties.  The current status of maintainers is largely unknown and has been for some time now.  Good governance requires that we remedy this with reasonable haste.

File: MAINTAINERS
Comment (GlennBullingham) [reply MAINTAINERS:3]: Sorry @vishnuitta but I don't think I understand what specific change it is that you are requesting.

Are you proposing that we include a statement to the effect that once maintainers have been transitioned to emeritus status, they cannot be removed from it (other than by restoration to active maintainer status)?  If so, then I think if enacted that would form a part of project governance policy and therefore should be added to the appropriate document(s), not the list itself.  That would make its treatment consistent with that for the election and withdrawal of other recognised roles.

Or do I misunderstand?

**Review by richardelling — APPROVED** [review]

**Review by mynktl — APPROVED** [review]

File: MAINTAINERS
Comment (vishnuitta) [reply MAINTAINERS:3]: you got it right @GlennBullingham . Can we have that project governance policy update some apt place in the repo?
All I'm trying to convey is: Lets make sure that due credits are given to the previous contributors even in the future.

Comment (GlennBullingham) [reply MAINTAINERS:3]: @vishnuitta I'm in complete and vehement agreement with you on that.  Due credit should given in perpetuity and to me at least, that's the purpose of the emeritus list(s), to call out those persons who have made valued contributions but have, as priorities change, ceased to actively engage.  In due course I will draft what I think to be an appropriate amendment / addendum on that matter to the current policy and solicit your review, that of the active maintainers (such as that list should end up, at the merging of this PR), and from all persons who contribute a review to this PR.

File: MAINTAINERS
Comment (kmova) [inline MAINTAINERS:54]: @akhilerm is still involved in the projects. 

A good way to check who is currently reviewing is to check the stats with last one year. 
https://openebs.devstats.cncf.io/d/75/prs-mergers-table?orgId=1&var-period_name=Last%20year&var-repogroup_name=All

**Comment by kmova** [comment]
@GlennBullingham - Inline with @vishnuitta - let us use the guidance provided here:
https://github.com/openebs/openebs/blob/main/GOVERNANCE.md#stepping-down-policy


As a step forward: 
- Can you please first update the governance to reflect the new policy? What changes need to be done?
- Use the data captured in the devstats and probably responses in the community slack or meetings to add or remove the folks from the owners file. 

I see that there are already 3 folks that have approved on this - probably - restrict the changes in this PR only to those three folks and open a new PR once the governance is updated.

**Review by kmova — CHANGES_REQUESTED** [review]
See comments above.

File: MAINTAINERS
Comment (pawanpraka1) [inline MAINTAINERS:10]: Is this the format for the following sections?

File: MAINTAINERS
Comment (pawanpraka1) [inline MAINTAINERS:44]: Can you put this list in alphabetical order?

**Review by pawanpraka1 — CHANGES_REQUESTED** [review]

**Comment by GlennBullingham** [comment]
Firstly, thank you everyone who has taken the time to either contribute a review and/or engage with me on this topic over other channels.

The need to formally establish a clearly defined membership of _active_ maintainers is, in my opinion, the most pressing matter as it is essential to the health and future growth prospects of the project. 

I take the suggestion above from @kmova as "a way to move forwards" with this necessary task, whilst taking into due consideration the changes requested by its reviewers.  Namely:

- Changes on _this_ PR will be restricted only to changes in the status of current project Maintainers.  The reviewer list will remain unchanged, subject to review in a later stage of this broader topic of governance updates/refreshes.
- Three current maintainers have approved this PR without requesting changes.  Those persons will be considered to have thus given their active consent to have their status changed from Maintainer to Emeritus Maintainer
- Those persons who have neither contributed a review, nor whom have I been able to elicit contact with via other channels are @ksatchit and @muratkars.  Since I believe it to be reasonable that recognition as an active Maintainer requires as a minimum for such persons to monitor for, and respond in a timely fashion to governance issues, and that for this purpose the project should hold valid contact details, the PR will move them to Emeritus Maintainer status.  As we have discussed above, the intention is that those persons holding such a status can be restored to full Maintainer status at any time, at their request by raising a PR here to that effect.  Emeritus status should be "for life", and I'll proposing that such concepts and procedures are incorporated into the formal governance policy.  I think that makes this extraordinary policy decision fair to all concerned.
- I have spoken with @gila directly and he has indicated that it is his wish to remain listed an active maintainer at this time.  I will include valid email details for him accordingly.
- Anyone who has contributed a request for changes will remain listed as an active maintainer at this time.  Subsequent PRs will seek to address review comments (i.e. updates to governance policy), and to call for a definitive yes/no from those persons as to whether they wish to remain *active* maintainers of the project.

**Comment by vishnuitta** [comment]
https://github.com/openebs/openebs/pull/3580#issuecomment-1297549757
Sounds good @GlennBullingham as you mentioned in this comment. I would like to look at the changes to make sure that I understood correct :)

**Comment by vishnuitta** [comment]
> * Use the data captured in the devstats and probably responses in the community slack or meetings to add or remove the folks from the owners file.

@GlennBullingham .. as @kmova suggested, it would be better to look at devstats and community responses as well to add or remove the people from the list. If you think that some XYZ person would be an active member in the future but not now, its always possible to come and update the list accordingly. It gives more data driven points for your proposal and the required recognition for those who are actively working on this project. (This need not be the only way but a good way to say that the proposed list is really active).

**Comment by GlennBullingham** [comment]
> [#3580 (comment)](https://github.com/openebs/openebs/pull/3580#issuecomment-1297549757) Sounds good @GlennBullingham as you mentioned in this comment. I would like to look at the changes to make sure that I understood correct :)

Hi @vishnuitta I've committed the changes discussed for review: https://github.com/openebs/openebs/pull/3580/commits/9ea42b5cf240f70a1cb79d6c22bc7aa18b82f67a

**Comment by GlennBullingham** [comment]
The scope of the changes effected by this PR now having been limited, as per the request of its reviewers, to just changes of status of specific (in)active maintainers (with either their explicit approval herein, or by merit of having being "timed out" unresponsive on available communications channels), and the proposed commit now having been available for review for the past 6 days, **I give fair warning that I will merge as-is on Friday 11th November after 5pm UK time, unless further comment requesting otherwise is made here before that time**.

**Review by kmova — APPROVED** [review]
Thank you @GlennBullingham for updating this! 

/lgtm!

**Review by pawanpraka1 — APPROVED** [review]
Looks good.

**Comment by GlennBullingham** [comment]
Thanks @kmova and @pawanpraka1.  Merging as described.

**Pull request merged**
